### DISCUSSION TITLE
es/guides/heatmaps/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#9744] [Docs]
# es/guides/heatmaps/

Visualización avanzada de datos con Ultralytics YOLOv8  Heatmaps

https://docs.ultralytics.com/es/guides/heatmaps/

**Comment by wisrovi** [comment]
tendrias el video de ejemplo que has usado?

**Reply by glenn-jocher** [reply]
Los videos de ejemplo específicos mostrados en la documentación no están disponibles para descarga directa. Sin embargo, puedes usar cualquier video para probar la funcionalidad de heatmaps con YOLO11:

```python
# Usar tu propio video
yolo solutions heatmap source="path/to/tu_video.mp4" show=True

# O usar videos de ejemplo disponibles públicamente
# Prueba con videos de tráfico, multitudes, o cualquier escena con movimiento
```

También puedes encontrar videos de prueba gratuitos en sitios como Pixabay o usar tu cámara web con `source=0` para testing en tiempo real.
